include/decoration.h

```diff
@@ -137,6 +178,18 @@ struct Decoration
     /*0x1c*/ const u16 *tiles;
 };
 
+struct DecoPCPointers
```

Comment:
This should be renamed DecorPCPointers or DecorationPCPointers, as the usual shorthand for decoration is decor.